check-disk/lib/check-disk.go

```diff
@@ -273,9 +273,10 @@ func listPartitions() ([]gpud.PartitionStat, error) {
 			"ignore":
 			continue
 		case "none":
-			if !strings.Contains(p.Opts, "bind") {
-				partitions = append(partitions, p)
+			if !isBindMount(p.Opts) {
+				continue
 			}
+			partitions = append(partitions, p)
```

Comment:
A mountpoint of fstype=none should be included if it is bind-mounted; otherwise ignored.

Ref: https://github.com/coreutils/gnulib/blob/c45faf7f48bcbe57df3bbf8df81a016c5f15df55/lib/mountlist.c#L182-L189
(This is the referenced source that the original code was introduced https://github.com/mackerelio/go-check-plugins/commit/4dce775e0c0d83583d25cc0cb699d2809324d81e)

Reply:
Maybe we don't need this at all?
https://github.com/coreutils/gnulib/blob/a742bdb3fd6f5ebaa730824eda7fcfe7faedb451/lib/mountlist.c#L541-L545

Reply:
In my opinion it might not be needed in case of reading files provided by procfs. But I think it is good to keep this check if you worry.